### PR TITLE
Allow a custom set of characters in a token

### PR DESCRIPTION
**Pull request by carpie**
Here is a proposed way to handle issue #86.

### CODE REVIEW

File: boolean/boolean.py [hunk 1]
```diff
@@ -114,7 +114,8 @@ class BooleanAlgebra(object):
     """
 
     def __init__(self, TRUE_class=None, FALSE_class=None, Symbol_class=None,
-                 NOT_class=None, AND_class=None, OR_class=None):
+                 NOT_class=None, AND_class=None, OR_class=None,
+                 allowed_in_token=('.', ':', '_')):
```
Comment (pombredanne) [inline boolean/boolean.py]: Could it be better to provide a string that lists all allowed characters explicitly (including "alnum")?

Comment (carpie) [reply boolean/boolean.py]: The idea is good, but maybe not practical to match what `alnum` allows?  According to the python docs, `isalpha` (part of `alnum`) matches any unicode character marked as 'Letter'.  I'm not sure how many that is exactly, but probably more than one would want to specify.

Comment (pombredanne) [reply boolean/boolean.py]: Good point. Using a string vs a list is minor too. So I am merging as-is.
Thank you again!

Comment (carpie) [reply boolean/boolean.py]: Thank you!

**Pull request merged**